coding-concepts.js

```diff
@@ -67,7 +71,9 @@ class Learn {
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
+
+// a) Your answer: This does not contain return, and so it will not return anything.
+// b) Verify and explain: This returns: Learn { student: 'George', cohort: 'Echo', year: 2022 }. I did some research and learned that contructors do not usually have a return statement! The constructor's task is to write the necessary components within itself, and it automatically becomes the result.
+
```

Comment:
Classes are blueprints or building blocks for objects. new is the keyboard to instantiate a class aka create a new instance of the class aka create an object. The constructor will show what properties each instance of this class will have. It also shows it that instantiation will need an argument. In the Learn class, the constructor is stating that a name needs to be passed when creating an object from this class.